appeals/web/src/server/appeals/appeal-details/interested-party-comments/redact/components/buttons.js

```diff
@@ -0,0 +1,47 @@
+import { wrapComponents } from '#lib/mappers/index.js';
+
+/** @typedef {import("#appeals/appeal-details/interested-party-comments/interested-party-comments.types.js").Representation} Representation */
+
+/** @type {PageComponent[]} */
+export const buttons = [
+	wrapComponents(
+		[
+			{
+				type: 'button',
```

Comment:
This might be a good time to add a `button` page component helper to `appeals/web/src/server/lib/mappers/components/page-components`, but happy to leave that for a separate PR if you think it would add too much to this one

Reply:
I think that's a good idea, happy to add it to this